src/fragments/lib/utilities/js/hub.mdx

```diff
@@ -7,6 +7,19 @@ import { Hub } from 'aws-amplify';
 
 ## Working with the API
 
+### Channels
+A channel is a logical group name that you use to organize messages and listen on. These are strings and completely up to you as the developer to define for dispatching or listening. However, while you can dispatch to any channel, ***Amplify protects certain channels*** and will flag a warning as sending unexpected payloads could have undesirable side effects (such as impacting authentication flows). The protected channels are currently:
```

Comment:
Can we change this paragraph slightly like so:

The hub module allows you to see different dispatched events in channels that represent the Amplify packages. You also can set your own custom channels. However, while you can dispatch to any channel, ***Amplify protects certain channels*** and will flag a warning as sending unexpected payloads could have undesirable side effects (such as affecting authentication flows). The protected channels are currently:

Reply:
As we discussed in Slack, let's just keep it as it is right now.